xchange-core/src/test/java/org/knowm/xchange/dto/account/FundingRecordStatusTest.java

```diff
@@ -11,35 +11,35 @@
 
   @Test
   public void shouldProcessStatusDescriptionNormal() throws Exception {
-    testStatusDesc("PROCESSING", "foo", FundingRecord.Status.PROCESSING, "foo");
+    testStatusDesc(FundingRecord.Status.PROCESSING, "foo", FundingRecord.Status.PROCESSING, "foo");
   }
 
   @Test
   public void shouldProcessStatusToUpercase() throws Exception {
-    testStatusDesc("Complete", "bar", FundingRecord.Status.COMPLETE, "bar");
+    testStatusDesc(FundingRecord.Status.COMPLETE, "bar", FundingRecord.Status.COMPLETE, "bar");
```

Comment:
This test ("shouldProcessStatus**ToUpercase**") makes no sense now; it should be removed, renamed or changed.

Reply:
You are right. I will address this